base/testfiles-lthooks/lthooks-026.lvt

```diff
@@ -6,13 +10,13 @@
 \START
 
 
-\BeforeBeginEnvironment{quote}{\typeout{quote begin}}
-\AtBeginEnvironment{quote}{\typeout{quote begin}}
-\AtEndEnvironment{quote}{\typeout{quote end}}
-\AfterEndEnvironment{quote}{\typeout{quote after}}
+\BeforeBeginEnvironment{quote}{\typeout{quote before}}% env/quote/before
+\AtBeginEnvironment{quote}{\typeout{quote begin}}%      env/quote/begin
+\AtEndEnvironment{quote}{\typeout{quote end}}%          env/quote/end
+\AfterEndEnvironment{quote}{\typeout{quote after}}%     env/quote/after
```

Comment:
better add (top-level) into the typeout. that makes understanding the resulting order in the test output much easier